app/utils/location.tsx

```diff
@@ -101,3 +103,20 @@ export const getLocation = () =>
   getLocationBrowser()
     .catch(() => getLocationGoogle())
     .catch(() => useDefaultSanFranciscoLocation());
+
+/**
+ * Renders address metadata based on the `addresses` array in the `hit_` object.
+ *
+ */
+export const renderAddressMetadata = (hit_: SearchHit): JSX.Element => {
```

Comment:
🤔 I'm curious why the parameter has an underscore. Is this a naming convention we use for some particular case?

Reply:
I think it's a parameter they were using but this helper function came with the project. I just moved it into this file to declutter the component it was in.